.travis.yml

```diff
@@ -2,14 +2,15 @@ language: groovy
 
 jdk:
     - oraclejdk8
-    
-before_install:
-    - sudo apt-get update -qq
-    - sudo apt-get install oracle-java8-installer
+
+# Use container build
+sudo: false
+
+before_script:
     - export GRADLE_OPTS='-Dorg.gradle.parallel=false'
```

Comment:
I think it was the Server tests that failed when it was done in parallel, as each server test now uses any available port instead of a specific one, I think we can try enabling it again.